### ISSUE TITLE
Communicate thet Timber v2 is more than a templating system for WordPress

### ISSUE THREAD

**Issue by szepeviktor** [timber/timber#2334]
> Timber helps you create fully-customized WordPress themes faster with more sustainable code.

Currently we suggest that Timber can be used only for templating thus outputting (HTML) content.

It would be nice to list in what aspects Timber v2 is more than that.

**Comment by acobster** [comment]
Good idea @szepeviktor! I completely agree. The big sell for us at SiteCrafting was not just that you could use Twig (although that was a significant factor) - it was that you could write in a much more OO style that encouraged a more disciplined separation of concerns. In particular, it allowed us to get much closer to the MVC pattern.

Of course WP still manages to find plenty of ways to break those abstractions  :stuck_out_tongue: so my only caveat would be against making absolute-sounding statements like, "Timber lets you write MVC code in WordPress."

**Comment by jarednova** [comment]
Absolute-sounding statements? But that's what marketing is all about!

**Comment by acobster** [comment]
@jarednova you're right, that's literally the only thing that marketing is about. What I actually meant to say is that we should **only EVER** make absolute statements!

**Comment by jarednova** [comment]
there we go, that's the spirit!!

**Comment by Levdbas** [comment]
I've added 2.0 and docs tags since I think this is something we really need to look into. A lot of hard went into the fine folks that worked on 2.x for so long, so a bit of marketing is deserved.

**Comment by gchtr** [comment]
Yes, I agree. But let this issue not be a blocker to release, let’s tackle this after the release.

**Comment by mmorris8** [comment]
The decoupling of template display logic from PHP backend Timber provides opened the door to faster and smoother functional tests using Playwright.  With a total of 178 tests across 10 devices Timber has proved a major aid to making the site work cleanly and meet accessibility targets.

**Comment by jarednova** [comment]
That's great to hear @mmorris8 ! Forgive me if I missed it, but what site are you referencing?

**Comment by mmorris8** [comment]
The site I work on for my employer. www.covenanthealth.com. The heavily tested version goes out tomorrow morning.

I also created a discussion thread on this some time ago: https://github.com/timber/timber/discussions/3093

**Comment by jarednova** [comment]
great to hear @mmorris8 — good luck w the launch!

**Comment by mmorris8** [comment]
I hit a hiccup yesterday but it's up as of this morning if anyone wants to take a look.

**Comment by JustinDouglas16** [comment]
> I hit a hiccup yesterday but it's up as of this morning if anyone wants to take a look.

I tried looking your site up but for some reason got 403 forbidden. Could be me but just wanted to point it out.
